utils.py

```diff
@@ -99,17 +99,39 @@ def arg_parser() -> argparse.Namespace:
         default=False,
     )
 
+    parser.add_argument(
+        "-s",
+        "--sim",
+        help="Runs the data simulation alongside the mock simulation, with an optional scale",
```

Comment:
Maybe down the line we might have to make this a subcommand so we can pass in more arguments to the sim.
So something like `python main.py sim --type full-scale --temperature 278`

Reply:
atp just change the config imo